docs/source/reference/yaml-spec.rst

```diff
@@ -347,12 +403,21 @@ OR
 ``resources.memory``
 ~~~~~~~~~~~~~~~~~~~~
 
-Memory in GiB per node (optional).
+Memory specification per node (optional).
 
 Format:
 
 -  ``<num>``: exactly ``<num>`` GiB
 -  ``<num>+``: at least ``<num>`` GiB
+-  ``<num><unit>``: memory with unit (e.g., ``32GiB``, ``64GB``)
+
+Units supported (case-insensitive):
+- B (bytes, 2^0 bytes)
+- K/KB (kilobytes, 2^10 bytes)
```

Comment:
I’d prefer to not support the single letter units as (1) they are less standard (2) too many variants supported (GB/GiB) already. Is it quick to remove it? 

In docs we should stick with either with our without i.